generic/bun/egg-bun.json

```diff
@@ -0,0 +1,122 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-08-27T01:47:23+02:00",
+    "name": "Bun",
+    "author": "mratox@protonmail.com",
+    "description": "Bun is an incredibly fast JavaScript runtime, transpiler bundler, and npm package manager all-in-one.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/mratox\/images:bun": "ghcr.io\/mratox\/images:bun"
```

Comment:
Please PR an image to my yolks repo. You will need to configure it in the build system as well

Reply:
Done